### ISSUE TITLE
xpath Issue

### ISSUE THREAD

**Issue by y0dev** [liamcollins/facebook_downloader_source#1]
I'm looking at the source code and it seems to keep crashing here. I'm not too familiar with XPath can you explain what is  the meaning of this XPath

![xpath image](https://user-images.githubusercontent.com/11625997/115084537-edc43b00-9ebd-11eb-8e36-7a633e859f9f.png)

Heres the error it's throwing
![image](https://user-images.githubusercontent.com/11625997/115084804-3bd93e80-9ebe-11eb-88f1-8ee296cc2ee2.png)

Exception thrown: 'OpenQA.Selenium.WebDriverTimeoutException' in WebDriver.dll
An unhandled exception of type 'OpenQA.Selenium.WebDriverTimeoutException' occurred in WebDriver.dll
Timed out after 5 seconds

**Comment by y0dev** [comment]
I fixed the issue.

**Comment by liamcollins** [comment]
Nice! Did you want to submit a pull request for your change? The problem with the way I set this up is that XPaths are super fragile and they break everytime FB changes something on the page.

**Comment by y0dev** [comment]
Yeah I would like to submit a PR for you to test. Hopefully it works on anyone's account. I closed my account after using it so I couldn't really unit test it. 

I also had to comment out that break in my own Checkout (this is not what I tried to push though) that gets triggered if the previous source matches the source which helped me collect all 900 photos of mine. That seemed like a small work around for the time being. Pretty sure there is a better way.
